File: template/app/src/main/java/com/xmartlabs/template/database/AppDataBase.java
package com.xmartlabs.template.database;

import com.raizlabs.android.dbflow.annotation.Database;

@Database(name = AppDataBase.NAME, version = AppDataBase.VERSION)
public class AppDataBase {
  //TODO: change db name
  public static final String NAME = "template_database";
  public static final int VERSION = 1;
}

File: template/app/src/main/java/com/xmartlabs/template/database/AppDataBase.kt
package com.xmartlabs.template.database

import com.raizlabs.android.dbflow.annotation.Database

@Database(name = AppDataBase.NAME, version = AppDataBase.VERSION)
object AppDataBase {
  //TODO: change db name
  const val NAME = "template_database"
  const val VERSION = 1
}
